Xamarin.PropertyEditing.Mac/Themes/MacThemeManager.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Drawing;
+using AppKit;
+
+namespace Xamarin.PropertyEditing.Themes
+{
+	public class MacThemeManager : BaseThemeManager
+	{
+		public NSAppearance CurrentAppearance { get; private set; }
+
+		NSAppearance DarkAppearance = NSAppearance.GetAppearance (NSAppearance.NameVibrantDark);
```

Comment:
There don't seem to be any of the colors defined by Vaclav present?

Reply:
As per my other comment regarding ThemeStyle, the 1st version I showed Vaclav used the ThemeStyle properties to style the control's backgrounds etc. But it wasn't 100% right and we would need to create custom controls, to achieve the styling. After speaking to him he said he wanted to styling to work like Xamarin Studio. This is exactly how XS achieves its styling, hence the change to dump the ThemeStyle code and use the NSAppearance mechanism above.

Reply:
My point isn't about the mechanism used, but that the appearance selected does not match his colors selected in the design. I assume there's a way to define a custom appearance and that he isn't intending that you ignore his theme.

Reply:
Custom themes are possible, but those were abandoned by the XS team due to the fact Apple have not provided a tool to create them and the unreliability of 3rd party tools, as told to me by Sevo Kukol from the XS team. 
Pinging Vaclav here... 
@vancura can you please confirm that we want the PropertyEditor to follow the Xamarin Studio manner of theming? This is what the Property Editor currently looks like following XS-Mac theming...

<img width="484" alt="screen shot 2017-10-09 at 16 47 45" src="https://user-images.githubusercontent.com/271363/31346780-24ca49c4-ad12-11e7-9e50-a8b52412d459.png">
<img width="487" alt="screen shot 2017-10-09 at 16 50 16" src="https://user-images.githubusercontent.com/271363/31346787-2a031038-ad12-11e7-9739-ebbd391894c2.png">

Reply:
Yes, please use XS theming. Colors defined in my comps are a part of future redesign, we'll get there, but for now use whatever we have in XS.

WRT your question about how we do it (/c @sevoku): we have a [file with all colors](https://github.com/mono/monodevelop/blob/master/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/Styles.cs), so it would be best if Property Pad used colors from here. Maybe I am wrong, I don't exactly know what's the best approach code-wise, but I believe this is the best we can do (@sevoku might think something else).

If you use `Styles` from `MonoDevelop.Ide` we'll be able to change the colors later to fit the rest of redesign.

Makes sense?

Reply:
I just have a comment about your two screenshots above; please take a look at the space between two columns: it seems you have both columns too close to each other, there needs to be a gap. Thanks!

Reply:
More like this...
<img width="484" alt="screen shot 2017-10-10 at 16 18 10" src="https://user-images.githubusercontent.com/271363/31394628-afeb5e16-add6-11e7-8def-dbec0d966b68.png">

Reply:
@CartBlanche The spacing is defined in his document at 12pt.

> but I believe this is the best we can do (@sevoku might think something else).

We can't use that directly for a variety of reasons, we'll discuss code handling in Slack.

Reply:
@vancura If we're going to use that color list, can you annotate the designs to call out what color titles each piece is using?

Reply:
Yes, but not now – I am too busy with other stuff before coming to Boston on Saturday. I think I should be able to find some time for it next week, between meetings.

Reply:
@vancura this is a screen shot from the conversation we had about what it would look like using the NSAppearance Dark theme. which is what I implemented and PRed. 
<img width="582" alt="screen shot 2017-10-11 at 15 33 11" src="https://user-images.githubusercontent.com/271363/31447305-969bc144-ae99-11e7-9a0f-f0101b1e6ee7.png">
Hence my confusion about why we now need to revert my original Styling change, which has lots of custom control implications, AFAICT. Maybe @sevoku can enlighten us further.

Maybe we should investigate custom NSAppearances again and get that working instead of the Styling property changes. Possibly fork an existing tool that does most of the custom NSAppearance heavy lifting and we can contribute fixes/enhancements so that the whole Apple community benefits from the enhanced tool, at least until Apple releases one??

I'll be pushing an update to fix the 12pixel separation issue shortly.

Reply:
I based the comps on dark `NSAppearance` (and light too), or rather how bad it is – apparently Apple forgets about it, and all we have is just a buggy implementation with a ton of workarounds (@sevoku would tell stories).

I tried to mix colors and shapes to replicate the feeling of Apple's treatment, to make it look at usable. The target was to have as little work as possible when mixing with other controls by Apple. And also to have just minimum custom controls and use what they give us much as possible.

So yes, this is what I thought would be the end if everything works well. But of course, if we can't have it for any reason, I am open to discussion. It doesn't mean I like what Apple does, and it was just the fastest way and the most compatible.